sessions/006/README.md

```diff
@@ -326,3 +326,256 @@ pvc-0bd38196-2f5b-4a05-8917-b43bd2dde50b   20Gi       RWO            Delete
+
+This procedure has some tricky steps highlighted in bold, where you need to be extra careful to avoid losing data.
+**Don't use Minikube for this example, as it doesn't have full volume support.**
+
+First, [deploy the Strimzi Cluster Operator](/sessions/001).
+For the sake of this example, we deploy the Kafka cluster reducing the disk size.
+
+```sh
+$ cat sessions/001/resources/000-my-cluster.yaml \
+    | yq ".spec.zookeeper.storage.size = \"1Mi\"" \
+    | kubectl create -f -
+kafka.kafka.strimzi.io/my-cluster created
+
+$ kubectl create -f sessions/001/resources/001-my-topic.yaml
+kafkatopic.kafka.strimzi.io/my-topic created
+```
+
+When the cluster is ready, break the zookeeper by creating 3 topics and 1000 partitions per topic, which exceeds the disk capacity of 1 MiB.
+
+```sh
+$ CLUSTER_NAME="my-cluster" \
+  ZK_PODS="$(kubectl get po -l strimzi.io/name=my-cluster-zookeeper --no-headers -o custom-columns=':metadata.name')" \
+  NEW_PV_CLASS="$(kubectl get pv | grep my-cluster-zookeeper-0 | awk '{print $7}')" \
+  NEW_PV_SIZE="100Mi"
+
+$ kubectl get pvc -l strimzi.io/name=$CLUSTER_NAME-zookeeper
+NAME                          STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS                           AGE
+data-my-cluster-zookeeper-0   Bound    pvc-7979a0d6-17e3-4ccc-adf5-72fd033073ee   1Mi        RWO            ocs-external-storagecluster-ceph-rbd   6m
+data-my-cluster-zookeeper-1   Bound    pvc-787734b0-c3e4-4068-a964-dc56802ccb65   1Mi        RWO            ocs-external-storagecluster-ceph-rbd   6m
+data-my-cluster-zookeeper-2   Bound    pvc-2250763e-0d8c-4e78-9c3b-5e00ec40970a   1Mi        RWO            ocs-external-storagecluster-ceph-rbd   6m
+
+
+$ kubectl-kafka bin/kafka-topics.sh --bootstrap-server $CLUSTER_NAME-kafka-bootstrap:9092 --topic t0 --create --partitions 1000 && \
+kubectl-kafka bin/kafka-topics.sh --bootstrap-server $CLUSTER_NAME-kafka-bootstrap:9092 --topic t1 --create --partitions 1000 && \
+kubectl-kafka bin/kafka-topics.sh --bootstrap-server $CLUSTER_NAME-kafka-bootstrap:9092 --topic t2 --create --partitions 1000
+Created topic t0.
+Created topic t1.
+Error while executing topic command : The request timed out.
+[2024-02-29 09:42:21,758] ERROR org.apache.kafka.common.errors.TimeoutException: The request timed out.
+ (kafka.admin.TopicCommand$)
+
+$ kubectl get po -l strimzi.io/name=$CLUSTER_NAME-zookeeper
+NAME                     READY   STATUS             RESTARTS        AGE
+my-cluster-zookeeper-0   0/1     CrashLoopBackOff   5 (2m16s ago)   16m
+my-cluster-zookeeper-1   0/1     CrashLoopBackOff   5 (2m24s ago)   16m
+my-cluster-zookeeper-2   1/1     Running            0               16m
+
+$ kubectl logs $CLUSTER_NAME-zoookeeper-0 | grep "No space left on device" | tail -n1
+java.io.IOException: No space left on device
+```
+
+Even if not all pods are failed, we still need to increase the volume size of all brokers because the storage configuration is shared.
```

Comment:
```suggestion
Even if not all pods are failed, we still need to increase the volume size of all servers because the storage configuration is shared.
```